SystemInfo/Value/ComposerPackage.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/**
+ * File containing the ComposerPackage class.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\EzSupportToolsBundle\SystemInfo\Value;
+
+use eZ\Publish\API\Repository\Values\ValueObject;
+
+/**
+ * Value for information about a Composer package.
+ */
+class ComposerPackage extends ValueObject implements SystemInfo
+{
+    /**
+     * Version.
+     *
+     * Example: v2.7.10
+     *
+     * @var string
+     */
+    public $version;
```

Comment:
How? Check if starting with "(v)0" or "dev-"? These versions seem somewhat inconsistent, the regexp for it may require some testing & maintenance. But I'm not against it.